recipes/openbabel/2.4.1/meta.yaml

```diff
@@ -35,12 +37,15 @@ requirements:
         - zlib
         - libxml2
         - cairo   # [linux]
+        - xorg-libxau
```

Comment:
See e.g. https://travis-ci.org/bioconda/bioconda-recipes/jobs/230372715 , where openbabel is a run requirement for rDock (and actually the reason for this PR).